webapp-django/crashstats/signature/templates/signature/signature_summary.html

```diff
@@ -1,6 +1,7 @@
-<div class="panel">
-    <header class="title">
+<div class="panel tab-inner-panel">
```

Comment:
It's confusing why this is needed everywhere? What's wrong with using the `.panel`?